app/scripts/directives/logViewer.js

```diff
@@ -256,7 +268,7 @@ angular.module('openshiftConsole')
               // Stop any active streamer.
               stopStreaming();
 
-              if (!$scope.name) {
+              if (!resourceName) {
```

Comment:
Not sure this check is needed here anymore. If resource is not set, it would fail when we use `resource.kind` above.